cabal-install/src/Distribution/Client/Errors.hs

```diff
@@ -211,6 +297,162 @@ exceptionMessageCabalInstall e = case e of
   UnknownUserConfigSubcommand extraArgs -> "Unknown 'user-config' subcommand: " ++ unwords extraArgs
   ManpageAction extraArgs -> "'man' doesn't take any extra arguments: " ++ unwords extraArgs
   UnrecognizedResponse -> "unrecognized response"
+  CheckTarget ->
+    "The 'fetch' command does not yet support remote tarballs. "
+      ++ "In the meantime you can use the 'unpack' commands."
```

Comment:
This one pre-existed but could be fixed as part of this: I think `unpack` has been officially superseded by `get`.

Reply:
Good call